src/Raven.Client/Documents/Linq/LinqPathProvider.cs

```diff
@@ -98,6 +98,17 @@ public Result GetPath(Expression expression)
                     return CreateCounterResult(callExpression);
                 }
 
+                if (IsMetadataCall(callExpression))
+                {
+                    return new Result
+                    {
+                        MemberType = callExpression.Method.ReturnType,
+                        IsNestedPath = false,
+                        Path = "getMetadata",
```

Comment:
I see that we do have a const for that although in a server side code:
https://github.com/ravendb/ravendb/blob/21648565af49a7bbcc9c6adf2e9765223b2b2b36/src/Raven.Server/Documents/Patch/ScriptRunner.cs#L224

I believe we have more such method names that we use in JS projections (can you check that?). I wonder if we shouldn't move them to `Raven.Client.Constants` as internal constants and use across the client and server side code. @aviv86 What do you think?

Reply:
Note: it's not for this PR - but we might create an issue for such refactoring

Reply:
good idea

Reply:
Please create a ticket for that